schema/messaging.proto

```diff
@@ -479,3 +479,24 @@ message SendBlurbsResponse {
   // The number of successful blurb creations.
   int32 success_count = 1;
 }
+
+// The request message for the google.showcase.v1alpha3.Messaging\Connect
+// method.
+message ConnectRequest {
+  message ConnectConfig {
+    // The room or profile to follow and create messages for.
+    string parent = 1 [
+      (google.api.resource_reference) = "BlurbParent",
+      (google.api.field_behavior) = REQUIRED
```

Comment:
Does it make sense to put `REQUIRED` on a field that must be sent iff the parent message is used?

Reply:
I do not understand the question. If parent can be empty, then it is not required. If parent must be non-empty, then it can be set as required.

Reply:
Bah, `parent` is overloaded in this context. I mean to say: the parent _field_ must be set IFF ConnectConfig is used in the oneof below.  but if the ConnectConfig is not used then the parent _field_ is not required. 

In some contexts the parent field is required and in some contexts it is not. I'm guessing `(google.api.field_behavior) = REQUIRED` will be too strict fort his specific case though.

Reply:
Correct. You should not include the `field_behavior` annotation at all in that case.